modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -2239,6 +2242,8 @@ private void body0() {
                         segmentSize.put(segIdx, zip.length());
                         segmentAware.addCurrentWalArchiveSize(zip.length());
 
+                        metrics.onWalSegmentCompressed(zip.length());
```

Comment:
we request `zip.length` three times in this block of code - please extract it to local variable and reuse